scripts/convert_publish.py

```diff
@@ -0,0 +1,227 @@
+                    "rewardweight": utils.get_prop_raw(doc["reward_weight"]),
+                    "state": messagestate,
+                    "childcount": doc["children"],
+                    "closed": isClosedMessage,
+                    "level": 0 if orphan_comment else doc["depth"], # this value will be incorrect for comment to orphan comment
+                                                                    # but we only use level for comments nesting limits, 
+                                                                    # so it seems that this is not a problem
+                    "_SCOPE_": doc["author"],
+                    "_PAYER_": "gls.publish",
+                    "_SIZE_": 50
+                }
+                self.publish_tables.message.append(message)
+                
+                tags = []
+                if (isinstance(doc["json_metadata"], dict)):
+                    if ("tags" in doc["json_metadata"]):
+                        tags = create_tags(doc["json_metadata"]["tags"])
+
+                if(isinstance(doc["json_metadata"], str)):                
+                    try:
+                        if (doc["json_metadata"]):
+                            json_str = doc["json_metadata"]
+                            if ((json_str.find("\"") == 0) and (json_str.rfind("\"") == len(json_str)-1)):
+                                json_str = json_str[1: len(json_str)-1]
+                            dict_metadata = json.loads(json_str)                    
+                            if (dict_metadata["tags"]):
+                                tags = create_tags(dict_metadata["tags"])
+                    except Exception:
+                        tags= []
+                
+                content = {
+                    "id": cur_mssg_id,
+                    "headermssg": doc["title"],
+                    "bodymssg": doc["body"],
+                    "languagemssg": "",
+                    "tags": tags,
+                    "jsonmetadata": doc["json_metadata"],
+                    "_SCOPE_": doc["author"],
+                    "_PAYER_": "gls.publish",
+                    "_SIZE_": 50
+                }
+                self.publish_tables.content.append(content)
+                
+                if added % self.cache_period == 0:
+                    print("messages converted -- ", added)
+                    self.publish_tables.writeCache()
+                added += 1
+                utils.printProgressBar(passed + 1, length, prefix = 'Progress:', suffix = 'Complete', length = 50)
+            except Exception as e:
+                print(doc)
+                print(e.args)
+                print(traceback.format_exc())
+        try:
+            self.publish_tables.writeCache()
+        except Exception as e:
+            print(e.args)
+            print(traceback.format_exc())
+        
+        pool["state"]["rshares"] = Decimal128(Decimal(rshares_sum))  # TODO: how should we store int128_t?
```

Comment:
As I can see, uint128_t saved in database as a string with hexadecimal value.
In database I see this data after start Golos smart-contracts and execute `setrules` action in publication smart-contract:
```
> db.rewardpools.find({},{state:1})
{ "_id" : ObjectId("5beaf8cde5d50a5e083178e5"), "state" : { "msgs" : NumberLong(0), "funds" : { "amount" : NumberLong(1546237), "decs" : NumberLong(6), "sym" : "GLS" }, "rshares" : "0x00000000000000000000000000000000", "rsharesfn" : "0x00000000000000000000000000000000" } }
```

Reply:
thx. done.